### PR TITLE
Fix wrong dates in calendar events imported from caldav

### PR DESCRIPTION
**Pull request by mardy**
Fixes https://github.com/ubports/ubuntu-touch/issues/760

### CODE REVIEW

**Comment by Flohack74** [comment]
Hmm from the code this looks good, anything else to know before we merge?

File: organizer/qorganizer-eds-engine.cpp [hunk 1]
```diff
@@ -1324,6 +1324,10 @@ QDateTime QOrganizerEDSEngine::fromIcalTime(struct icaltimetype value, const cha
         } else {
             // fallback: sometimes the tzId contains the location name
             if (!timezone) {
+                static const char prefix[] = "/freeassociation.sourceforge.net/Tzfile/";
+                if (strncmp(tzId, prefix, sizeof(prefix) - 1) == 0) {
```
Comment (mariogrip) [inline organizer/qorganizer-eds-engine.cpp:1328]: I like the style without curly braces here, but its ok.

**Review by mariogrip — APPROVED** [review]
LGTM

**Comment by mariogrip** [comment]
Awesome :+1:

**Pull request merged**